pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraMergeCandidatesTest.scala

```diff
@@ -41,6 +40,17 @@ class SierraMergeCandidatesTest
         physicalAndDigitalSierraMergeCandidate(mergeCandidateBibNumber)
     }
 
+    it("trims spaces at the end of the bnumber of the mergeCandidate") {
+      val sierraData = createSierraBibDataWith(
+        varFields = create776subfieldsWith(
+          ids = List(s"(UkLW)$mergeCandidateBibNumber ")
+        )
+      )
+
+      SierraMergeCandidates(sierraData) shouldBe
+        physicalAndDigitalSierraMergeCandidate(mergeCandidateBibNumber)
+    }
```

Comment:
Might be worth a comment that this is based on real Sierra data, possibly including an example.